test/lib/tasks_test.rb

```diff
@@ -7,6 +7,7 @@ class TasksTest < ActiveSupport::TestCase
     travel_to "2016-12-19T19:00:00Z" do
       user = create(:user)
 
+      Rails.application.load_tasks
```

Comment:
Without this, I couldn't get the tests to pass locally :(

Reply:
Bummer 😕  I wonder why that is. I have no problems running them locally.

Still this is pretty harmless 👍